### DISCUSSION TITLE
How to use my electrodes position acquired by digitizer?

### DISCUSSION THREAD

**Discussion by npdrbong** [simnibs/simnibs#264] [7.Creating head models]
Hello experts at SIMNIBS,

I am a student who has recently become interested in brain stimulation. Recently, our lab purchased a Soterix HD-tDCS machine, and we plan to use it to measure and compare brain waves before and after brain stimulation. However, my question is about how to ensure that the stimulation location I desire is accurate.

In our lab, we use the Polhemus Fastrak equipment and CURRY 7 software as our navigation system. With this system, we can receive the position of each electrode or position (we plan to use Soterix's HD-cap) in a personalized coordinate as a .pom file. I found that Brainstorm3 can convert a .pom file into a channelmat, but I wonder what steps should be taken to use this data in SIMNIBS?

Currently, we have not measured the MRI of the subjects, but if necessary, we are considering taking MRI scans of the subjects and performing co-registration. I would appreciate any recommended workflow or documents for reference to study further.

**Comment by Kstoffer** [comment]
We don't have explicit support for .pom files and we do not really have knowledge of the format readily available. However, if you can export coordinates in the space of the head model you intend to use it should be fairly easy to write a script to use the coordinates, just take a look at https://simnibs.github.io/simnibs/build/html/tutorial/scripting.html.
By the way would the head model you intend to use be ernie or some other canonical head model given you don't have MRI scans?

**Reply by npdrbong** [reply]
Thank you for kind reply! I want to use my own MRI scan, creating own Head Model. In this case, because cap location/coordinates is not exact, I want to make new cap/electrode location/coordinate file with digitizer.

**Reply by jduemose** [reply]
If you have digitized electrode positions and are able to somehow read those (e.g., using brainstorm) then you will have to save them in a format that simnibs can read which would be a .csv file looking like this (here just some random data)
```
Electrode,-82.38965687661117,-16.895583942095623,-9.322330038174762,T7
Electrode,-81.041656362012176,-14.235080219010651,26.311563257105526,C5
Electrode,-67.72212712722029,-11.278378194600045,61.153962198662661,C3
Electrode,-38.336397997318869,-9.1381183977875242,86.436857561054339,C1
ReferenceElectrode,0.036464278783968559,-8.2215173688495238,97.345154973377007,Cz
Electrode,38.523328396893881,-9.1291631001169922,86.782149547593676,C2
...
```
i.e., columns of type, x, y, z, name. You will have to register your electrode locations to the MRI space yourself (e.g., by matching fiducials).

Hope this helps.

Jesper
